content/guide/_index.md

```diff
@@ -8,145 +8,207 @@ Welcome! This guide will help you get started with a Hugo Theme, including how t
+### Step 1: Install Hugo
 
-### Setting up local development
-
-The basic process of setting up local development is roughly the same in every theme:
-
- 1. Open a command prompt on the theme folder. (unpacked zip folder/themes/theme folder)
- 2. Run `cd exampleSite/`on command line to navigate the exampleSite folder.
- 3. Then run `hugo server --themesDir ../..` to serve this theme on your localhost.
- 4. Open your browser to your local server's address. (i.e. open Chrome to `localhost:1313`)
- 5. Edit source files and preview changes instantly with live reload.
-
-#### Install Hugo Extended version
-
-`hugo-extended` version is compulsory if you want to compile `SASS` or `SCSS`
+You need to install the `hugo-extended` version, if you want to compile `SASS` or `SCSS`
 
 * Homebrew (macOS)
-``` bash
+```bash
 brew install hugo
 ```
 
 * MacPorts (macOS)
-``` bash
+```bash
 port install hugo
 ```
 
 * Homebrew (Linux)
-``` bash
+```bash
 brew install hugo
 ```
 
 * Chocolatey (Windows)
-``` bash
+```bash
 choco install hugo-extended -confirm
 ```
 
 * Scoop (Windows)
-``` bash
+```bash
 scoop install hugo-extended
 ```
 
-#### Local development in a nutshell
+{{% notice note %}}For more information about other installation methods check out the [Hugo Documentation](https://gohugo.io/getting-started/installing/).{{% / notice %}}
 
-{{< youtube dyZcRRWiuuw >}}
+### Step 2: Initiate Hugo website
 
-**If the list above sounds like a foreign language to you, that's okay! Let's break it down:**
+Next we create a skeleton Hugo website.
 
- * **`Hugo` is a static site generator**. If you don't have Hugo installed, you'll need to [download](https://github.com/gohugoio/hugo/releases) and [install](https://gohugo.io/getting-started/installing/) it [make sure you have downloaded the [Hugo Extended](#install-hugo-extended-version) version for compiling scss files].
+```bash
+hugo new site my-hugo-site
+```
 
- * **A local server** is a temporary server environment on your computer-generated by your theme's build tool. This avoids browser limitations of serving files directly from your computer's filesystem (i.e. [CORS](https://developer.mozilla.org/en-US/docs/Glossary/CORS) limitations).
+### Step 3: Set up a Git repository
 
- * **Live reload** is a convenience function that many themes will include as part of their build tooling. It simply means that when you edit source files after they are recompiled, your browser will magically refresh and display the changes without a manual refresh.
+While this is not a mandatory requirement, we advise to set up a repository with your website code in it. This will make updates of your theme easier, and you have a backup of older versions of your website in case you mess up things when trying out new features.
```

Comment:
Codacy found an issue: [[maximum-line-length] Line must be at most 80 characters](https://app.codacy.com/gh/gethugothemes/gethugothemes-docs/pullRequest?prid=6304190)